Makefile.third

```diff
@@ -19,16 +19,18 @@ $(FREETYPE_LIB) $(FREETYPE_DIR)/include: $(THIRDPARTY_DIR)/freetype2/CMakeLists.
 		-DCXXFLAGS="$(CXXFLAGS)" -DLDFLAGS="$(LDFLAGS)" -DCHOST=$(CHOST) \
 		$(CURDIR)/$(THIRDPARTY_DIR)/freetype2 && \
 		$(MAKE)
-	cp -fL $(FREETYPE_DIR)/$(if $(WIN32),bin,lib)/$(notdir $(FREETYPE_LIB)) $@
+	cp -fL $(FREETYPE_DIR)/$(if $(WIN32),bin,lib)/$(notdir $(if $(DARWIN),libfreetype.6.dylib,$(FREETYPE_LIB))) $@
```

Comment:
Doesn't FREETYPE_LIB already evaluates to $(OUTPUT_DIR)/libs/libfreetype.6.dylib on OSX?